### PR TITLE
[Notebook] Link to snapshot should not be a fully qualified url #3445

### PR DESCRIPTION
**Pull request by nikhilmandlik**
### Resolves: 
resolves: #3445

### Author Checklist:
Check | Passed
-- | --
Changes address original issue? | Y
Unit tests included and/or updated with changes? | N/A
Command line build passes? | Y
Changes have been smoke-tested? | Y
Testing instructions included? | Y

### CODE REVIEW

File: src/plugins/notebook/components/NotebookEmbed.vue [hunk 1]
```diff
@@ -143,7 +143,9 @@ export default {
                 this.openmct.notifications.alert(message);
             }
 
-            window.location.hash = hash;
+            const relativeHash = hash.slice(hash.indexOf('#'));
```
Comment (nikhilmandlik) [inline src/plugins/notebook/components/NotebookEmbed.vue:146]: older version of notebook used to save this as full url, so need to extract only hash part for backwards compatibility.

**Comment by davetsay** [comment]
Hate to be the one to say this, but should we include unit tests for this?

**Comment by nikhilmandlik** [comment]
> Hate to be the one to say this, but should we include unit tests for this?

test related to this change mostly covers routing.

**Comment by shefalijoshi** [comment]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N but it will be as part of another PR
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue)

**Review by shefalijoshi — APPROVED** [review]

**Pull request merged**